dftimewolf/lib/processors/turbinia_base.py

```diff
@@ -460,10 +516,11 @@ def TurbiniaWait(self,
     retries = 0
     processed_paths = set()
     status = 'running'
+    wait_status = ['running', 'pending']
```

Comment:
We could move this to a module-level frozenset, no?

Reply:
Yep. Done.